tests/Http/Place/ImportPlaceRequestHandlerTest.php

```diff
@@ -143,7 +144,9 @@ protected function setUp(): void
             ),
             new CallableIriGenerator(fn ($placeId) => 'https://io.uitdatabank.dev/places/' . $placeId),
             $this->commandBus,
-            $this->imageCollectionFactory
+            $this->imageCollectionFactory,
+            true,
+            $this->createMock(LookupDuplicatePlace::class),
```

Comment:
This can use some extra unit tests and the feature test can be removed.

Reply:
Wrote the extra tests (feature test is in a separate pr, and I think both are useful)